### PR TITLE
Adds support for .env file variables using dotenv.

### PR DESCRIPTION
**Pull request by ncrmro**
Might look over the google auth to see if it works.

#132

### CODE REVIEW

**Comment by mattkrick** [comment]
this looks super! i'll be in a good internet zone tomorrow & can clone this & give it a go, thank you!

File: .env.example [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+GOOGLE_SECRET="mJD4yU2m_Zms3a"
+GOOGLE_CALLBACK_URL="http://localhost:3000/auth/google/callback"
+
+DATABASE_HOST=localhost
+DATABASE_PORT=28015
+DATABASE_SSL=false
+
+PROTOCOL=http
+HOST=localhost
+PORT=3006
```
Comment (mattkrick) [inline .env.example]: change to 3000

**Comment by mattkrick** [comment]
can you change the `.env.example` to just `.env`? otherwise we'll get a silent error since that's what it's looking for.

Google Oauth is broken, but it was half-broken before, so no worries, we can fix that later.  

The `NODE_ENV` in the `.env` will now override what we send it via CLI. To enable a single `.env` and allow for nice things like `npm run prod` I'm thinking we should remove it from the `.env`. Thoughts?

**Comment by ncrmro** [comment]
I concur about the NODE_ENV, and whoops on the port. I was under the assumption that you don't want the the .env in git and would rename the the .env.example on creation.

**Comment by mattkrick** [comment]
yeah, i say we include the `.env` in git so folks can get up & running without thinking if they just wanna see how it works. Make it a friendlier to beginners (but not too friendly 😄 ).

**Comment by ncrmro** [comment]
Definitely :+1:

**Pull request merged**